index.js

```diff
@@ -187,6 +188,15 @@ module.exports = {
 
   // After build output is ready, create a Percy build and upload missing build resources.
   outputReady: function(result) {
+    if (process.env.EMBER_CLI_TEST_COMMAND === 'true') {
```

Comment:
Do not call createPercyBuild when `ember build` is being run in the test evironment.